gdal/swig/python/osgeo/utils/rgb2pct.py

```diff
@@ -201,17 +160,16 @@ def main(argv):
                              ct,
                              callback=gdal.TermProgress_nocb)
 
-    tif_ds = None
```

Comment:
same as above. Shouldn't be moved before the gdal.Open() case

Reply:
same.